js/app.js

```diff
@@ -63,12 +71,21 @@ vote.photoArray.push(new Photo('img/kittens/cat14.jpg'));
 
 var lphoto = document.getElementById('leftphoto');
 var rphoto = document.getElementById('rightphoto');
+var results = document.getElementById('results');
```

Comment:
I'd have like to use jQuery more. I was excited about it. It was harder than it looked to "translate" JS to jquery. I almost don't even remember what was going on at this point, but do somewhat recall Sam saying even if we only used it a couple of times, that was good. I'd love to use it almost exclusively, as it seems so much concise.